compiler-rt/lib/scudo/standalone/common.cpp

```diff
@@ -12,13 +12,19 @@
 
 namespace scudo {
 
-uptr PageSizeCached;
+#if !defined(SCUDO_PAGE_SIZE)
```

Comment:
Not sure if there's the function ordering issue here. Can we group the cases of SCUDO_PAGE_SIZE together like
```
#if defined(SCUDO_PAGE_SIZE)
...
#else
...
#endif
```

Reply:
Do you mean pulling in the cases from common.h here? If so, I wanted to leave the getPageSizeSlow in the header to make it inline.

I did add a comment for getPageSize to indicate where it's coming from. I also modified linux.cpp so that getPageSize will not exist if the page size is hard-coded.

Reply:
Sorry I didn't notice that it's common.cpp and common.h (I thought this changes were in the same file). Please ignore this.